ietf/templates/base/menu.html

```diff
@@ -270,6 +270,12 @@
             Important dates
         </a>
     </li>
+    <li>
```

Comment:
This, like request a session below it, really needs to be conditional.
If someone _cant_ request a session or an interim, it is wrong to add this link. 
Unfortunately, this is being done in the base menus, and unless we're already paying the price to calculate whether the current requester can do these things, the most we should do is switch them on "logged in" to avoid making page loads even slower.
In the long run, this is something we can rework with with async population of the menu, but we're just beginning to build the scaffolding for that kind of thing.

So, for now, I'm of mixed mind. Do we continue down the current path of providing menu entries that are ultimately useless for most users, or do we stop now and make this _slightly_ better.

Reply:
Agreed. There are not many users with permissions to request interims. I think it is fair to expect that those are familiar enough with the DT to find the link on their group page.